frontend/client/App.js

```diff
@@ -8,43 +8,59 @@ import ManageTeams from './src/screens/ManageTeams'
 import Footer from './src/components/Footer'
 import NavBar from './src/components/NavBar'
 import NotFound from './src/screens/404'
+import Test from './src/screens/Test'
 import * as ROUTES from './src/constants/routes'
 import { createStore } from './src/store'
+import Container from 'react-bootstrap/Container'
+import Row from 'react-bootstrap/Row'
```

Comment:
I imported each component from react-bootstrap individually as it said to do so in the docs for react-bootstrap:  
  
"You should import individual components like: react-bootstrap/Button rather than the entire library. Doing so pulls in only the specific components that you use, which can significantly reduce the amount of code you end up sending to the client."  
  
https://react-bootstrap.github.io/getting-started/introduction#importing-components